internal/service/resourcepolicy/model_test.go

```diff
@@ -151,3 +150,24 @@ func Test_NewUserMetadataObjectTypeWithNilArg(t *testing.T) {
 	unit.AssertDiagsOK(t, diags)
 	assert.Equal(t, types.ObjectNull(resourcepolicy.UserMetadataObjectType.AttrTypes), obj)
 }
+
+func Test_NewTFPoliciesModelToSDK(t *testing.T) {
+	ctx := context.Background()
+	policies := []resourcepolicy.TFPolicyModel{
+		{
+			Body: types.StringValue("policy1"),
+			ID:   types.StringValue("id1"),
```

Comment:
is the ID necessary in the TF config to create a policy? isn't this returned by the API?

Reply:
Correct, doesn't really matter, shouldn't fail, changed on of the cases to not include `ID` in c1b1adb5d07aad9d03b330d76b42d7ef7a5c77a0